distributed/scheduler.py

```diff
@@ -130,54 +131,52 @@
 
 
 class ClientState:
-    """
-    A simple object holding information about a client.
+    """A simple object holding information about a client."""
 
-    .. attribute:: client_key: str
+    #: A unique identifier for this client. This is generally an opaque
+    #: string generated by the client itself.
+    client_key: str
```

Comment:
I added ``:members:`` to ``worker-memory.rst``. This means that attributes, methods, and properties that have a docstring and don't start with _ are now automatically documented.